psiminer-core/src/main/kotlin/Pipeline.kt

```diff
@@ -18,7 +18,7 @@ class Pipeline(
     private val repositoryOpener: PipelineRepositoryOpener,
     psiTreeTransformations: List<PsiTreeTransformation>,
     private val filters: List<Filter>,
-    val labelExtractor: LabelExtractor,
+    val labelExtractor: LabelExtractor<Any>,
```

Comment:
Same thoughts as in configs here, the pipeline doesn't need to know anything about label extractor generic. Use star-projection here too
```kotlin
val labelExtractor: LabelExtractor<*>
```